woocommerce/woocommerce
# add docker:ssh to core repo scripts

### All Submissions:


### Changes proposed in this Pull Request:

This adds `npm run docker:ssh` to this repo. Without this script you need to run `npm run docker:ssh` from the `tests/e2e/env` folder.

### How to test the changes in this Pull Request:

1. `npm run docker:up` and wait 30 seconds 
2. `npm run docker:ssh` 
3. `cat wp-config.php`
4. Verify WP config details are for your docker container.

### Other information:




### Changelog entry

N/A
